src/schema/artwork/__tests__/artwork.test.js

```diff
@@ -1194,4 +1194,201 @@ describe("Artwork type", () => {
       })
     })
   })
+
+  describe("#framed", () => {
+    const query = `
+      {
+        artwork(id: "richard-prince-untitled-portrait") {
+          framed {
+            label,
+            details
+          }
+        }
+      }
+    `
+    it("is null when framed is null", () => {
+      artwork.framed = null
+      return runQuery(query, rootValue).then(data => {
+        expect(data).toEqual({ artwork: { framed: null } })
+      })
+    })
+    it("is null when framed is false", () => {
```

Comment:
Had a minor comment in https://github.com/artsy/metaphysics/pull/1230/files#r212498908 here, too.